### ISSUE TITLE
Get it listed on the Gatsby plugins site under CMS

### ISSUE THREAD

**Issue by khalwat** [craftcms/gatsby-source-craft#36]
It'd be good to have representation here:

https://www.gatsbyjs.com/plugins/#cms

![Screen Shot 2021-05-28 at 10 31 21 AM](https://user-images.githubusercontent.com/7570798/120008167-b068c800-bfa8-11eb-9716-17672bd3cec1.jpeg)

...under the CMS category so people know it's an option.

**Comment by mattstein** [comment]
Agreed! This is on our radar, but we need a [perpetual project with Willit.build benchmarks](https://support.gatsbyjs.com/hc/en-us/articles/360052503494-Developing-a-first-class-CMS-integration-for-Gatsby-Cloud#step-6-documentation-required) we haven’t built yet.
